src/client/features/view/components/tooltips/metadataTip.js

```diff
@@ -22,15 +22,20 @@ class MetadataTip {
 
     //Get HTML for current data pair based on key value
     if (key === 'Standard Name') {
-      return h('div.fake-paragraph', h('div.field-name', 'Approved Name: '), pair[1].toString());
+      return this.makeTooltipItem(pair[1], 'Approved Name: ');
```

Comment:
Is there any reason why this is not a react component?

Reply:
class MetadataTip?

Reply:
@maxkfranz  @d2fong  

The MetadataTip Class is just a regular class which manages tooltips, to help avoid random DOM elements remaining in the document and to make binding tooltips to cytoscape-popper easier. 

The initial intention was that since tippy.js takes in only HTML elements or a DOM selector, it would make more sense to use hyperscript to generate HTML elements and pass them into tippy. Then tippy would manage rendering elements and the MetadataTip class would ensure they didn't lurk around and take up too much space. 

In this case where each tooltip is short-lived and rendered outside the react tree by tippy.js anyway, is there a benefit to making the class MetadataTip a react component?